### PR TITLE
chore: Update requirements

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Try updating to the newist version of brownie/other pakcages.  Maybe we can refork off the current version, which still seems to be getting updates/new chains.

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Note that if this is really working, we should probs tag the current brownie version or something and then refork it so we can add new chains that don't exist yet.  A bit unclear on how to do that as many things using the old style are pointing at the master branch of that fork.  Maybe we keep 1.20 on a new branch, or maybe make a new repo?

Thoughts? @gosuto-inzasheru

**Comment by Tritium-VLK** [comment]
This requires a fix to bal_addresses is_address.  Will focus on getting that merged and releasing 0.9.4 and then finish this up.

**Comment by Tritium-VLK** [comment]
ok @gosuto-inzasheru @jalbrekt85 as you can see this is working now.  In order to use new version of brownie (web3) have to deal with some function names changing.  bal_addresses has the utils to manage that.

I need to add the new chains.  I will add them to this new brownie.

I think my plan is to leave BalancerMaxis/brownie@master running the current version for now.  Then create a @1.20.x main and use that for the new version with new chains and start to update requirements where needed.  When we're sure almost everything is cutover from the old web3 naming/brownie versions we could consider merging the 1.20.x branch to main. 

Make sense?

Let's merge this step as is if it looks ok, just becase it's kind of the baseline.

**Review by gosuto-inzasheru — APPROVED** [review]

File: action-scripts/brownie/requirements-actions.txt [hunk 1]
```diff
@@ -1,12 +1,13 @@
-##eth-brownie==1.19.3
-git+https://github.com/BalancerMaxis/bal_addresses@0.9.1
-git+https://github.com/BalancerMaxis/brownie@chore/bump_black
-prettytable==3.7.0
-dotmap==1.3.30
-pathlib==1.0.1
+eth-brownie==1.20.3
+git+https://github.com/BalancerMaxis/bal_addresses@0.9.4
+#git+https://github.com/BalancerMaxis/brownie@chore/bump_black
```
[the remark this answers is not in the document]
Comment (gosuto-inzasheru) [reply action-scripts/brownie/requirements-actions.txt:3]: or are you leaving this line here on purpose..?

[the remark this answers is not in the document]
Comment (Tritium-VLK) [reply action-scripts/brownie/requirements-actions.txt:3]: I just figured i'd comment instead of delete and it could go later..

**Review by gosuto-inzasheru — APPROVED** [review]

**Pull request merged**